packages/ot-cryptid/PKGBUILD

```
@@ -0,0 +1,45 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+_name="OneTrick-CRYPTID"
+pkgname=ot-cryptid
+pkgver=1.0.1
+pkgrel=1
+pkgdesc='FM drum synthesizer with a Yamaha DX-7 emulation sound engine'
+arch=(aarch64 x86_64)
+url='https://punklabs.com/ot-cryptid'
+license=(GPL3)
+depends=(gcc-libs glibc libglvnd libx11 libxcb libxcursor xcb-util-wm)
+makedepends=(alsa-lib cargo jack python rust)
+optdepends=(
+  'jack: for running the stand-alone app'
+  'clap-host: for loading the CLAP format plugin'
+  'vst3-host: for loading the VST3 format plugin'
+)
+groups=(clap-plugins pro-audio vst3-plugins)
+source=("https://punklabs.com/content/projects/$pkgname/downloads/$_name-$pkgver-Source.zip")
```

```suggestion
source=("$pkgname-$pkgver.zip::https://punklabs.com/content/projects/$pkgname/downloads/$_name-$pkgver-Source.zip")
```
this makes it slightly easier to adapt if upstream later changes their file naming scheme